fastapi_login/__init__.py

```diff
@@ -1,5 +1,3 @@
 from .fastapi_login import LoginManager
 
-__all__ = [
-    LoginManager
-]
+__all__ = ("LoginManager",)
```

Comment:
Good eye catching that `__all__` should be used with a string and not the object directly.
Can you change the tuple back to a list? I have never seen `__all__` used with a tuple, the examples in the python docs also all use a list.

Reply:
Okay. But I still prefer tuple since it implies immutability.